src/components/comment-placer/fade-transition/FadeTransition.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { CSSTransition } from 'react-transition-group';
+
+import styles from './FadeTransition.css';
+
+export default class FadeTransition extends Component {
+    static propTypes = {
+        children: PropTypes.element.isRequired,
+        duration: PropTypes.number,
```

Comment:
I don't think this should be a property because the duration is hard-coded into the CSS file. Still, for consistency with the FadeAndGrow component, I agree to place it here if we it's actually used, like so `style={ { transitionDuration: `${duration}ms` } }`

Reply:
Got it. Will make it both hard-coded.